test.py

```diff
@@ -0,0 +1 @@
+print("Cam learns git!!!!")
```

Comment:
Please use full names.

```suggestion
print("Cameron learns git!!!!")
```